lib/model/store.dart

```diff
@@ -182,6 +187,41 @@ class PerAccountStore extends ChangeNotifier {
     } else if (event is AlertWordsEvent) {
       debugPrint("server event: alert_words");
       // We don't yet store this data, so there's nothing to update.
+    } else if (event is RealmUserAddEvent) {
+      debugPrint("server event: realm_user/add");
+      users[event.person.userId] = event.person;
+      notifyListeners();
+    } else if (event is RealmUserRemoveEvent) {
+      debugPrint("server event: realm_user/remove");
+      users.remove(event.userId);
+      notifyListeners();
+    } else if (event is RealmUserUpdateEvent) {
+      debugPrint("server event: realm_user/update");
+      final user = users[event.userId];
+      if (user == null) {
+        return; // TODO log
+      }
+      if (event.fullName != null)       user.fullName                   = event.fullName!;
+      if (event.avatarUrl != null)      user.avatarUrl                  = event.avatarUrl!;
+      if (event.avatarVersion != null)  user.avatarVersion              = event.avatarVersion!;
+      if (event.timezone != null)       user.timezone                   = event.timezone!;
+      if (event.botOwnerId != null)     user.botOwnerId                 = event.botOwnerId!;
+      if (event.role != null)           user.role                       = event.role!;
+      if (event.isBillingAdmin != null) user.isBillingAdmin             = event.isBillingAdmin!;
+      if (event.deliveryEmail != null)  user.deliveryEmailStaleDoNotUse = event.deliveryEmail!;
+      if (event.newEmail != null)       user.email                      = event.newEmail!;
+      if (event.customProfileField != null) {
+        if (user.profileData == null) {
+          return;
+        }
```

Comment:
Trying to set `user.profileData` with `{}` is getting flagged by the analyzer because it's `final`.

Its suggestion is to make it `late final`; is that OK?

Reply:
Right, it'd go with making it non-`final`. This is a mutable class in general — the rest of the fields aren't `final` — so that's fine.

Reply:
> This is a mutable class in general — the rest of the fields aren't `final` — so that's fine.

Sure. I think it's reasonable to prefer making properties that are `Map`s `final`, so that all the code using the model will definitely be working with the same `Map` instance.

But here there's this performance thing that involves not making it `final`, and anyway we're not thinking of introducing multiple `Map` instances, we just want to introduce one instance in the case where we don't have one yet.